src/popup/pages/profile.tsx

```diff
@@ -8,7 +8,7 @@ import { getUserData, getUserPRData, getUserHighlightsData } from "../../utils/f
 import { emojify } from "node-emoji";
 import { goBack, goTo } from "react-chrome-extension-router";
 import { getRelativeDays } from "../../utils/dateUtils";
-import { countUniqueRepos} from "../../utils/getContributedRepos";
+import { countUniqueRepos, PRResponse } from "../../utils/getContributedRepos";
```

Comment:
nit: https://github.com/open-sauced/ai/blob/beta/src/ts/types.ts#L21

Try doing it like this, not necessary to separate it into another file, but the naming can be changed.

Reply:
We can do this later anyway, thanks for the contribution!